formats/json-compact-refract.md

```diff
@@ -1,9 +1,10 @@
-# Compact Refract
+# JSON Compact Refract
 
-Compact Refract is a serialization format Refract for the purpose of removing a
-lot of the object keys that are repeated throughout the full serialization of
-Refract, as seen in the Refract specification. It also allows for expressing
-structures in a tuple, which resembles other formats like XML or Lisp.
+JSON Compact Refract is a serialization format Refract for the purpose of
+removing a lot of the object keys that are repeated throughout the full
+serialization of Refract, as seen in the Refract specification. It also allows
+for expressing structures in a tuple, which resembles other formats like XML or
+Lisp.
```

Comment:
What I was saying is that this PR should probably fix the compact refract spec too since the spec already exists and we don't have to write anything new. An example of things that need to be fixed:

> Ambiguity arises when an array is found in areas such as attributes or content and requires out-of-band information how to handle the array